test_vm/tests/evm_test.rs

```diff
@@ -231,34 +239,46 @@ fn test_evm_create() {
 }
 
 #[test]
-fn test_evm_eth_create_external() {
+fn evm_eth_create_external() {
     let store = MemoryBlockstore::new();
     let v = TestVM::<MemoryBlockstore>::new_with_singletons(&store);
 
+    evm_eth_create_external_test(&v, &v);
+}
+
+// TODO: drop usage of the concrete TestVM
+fn evm_eth_create_external_test<BS: Blockstore>(
+    v: &dyn VM<BS>,
+    _v_concrete: &TestVM<MemoryBlockstore>,
+) {
     // create the EthAccount
     let eth_bits = hex_literal::hex!("FEEDFACECAFEBEEF000000000000000000000000");
     let eth_addr = Address::new_delegated(EAM_ACTOR_ID, &eth_bits).unwrap();
     apply_ok(
-        &v,
+        v,
         &TEST_FAUCET_ADDR,
         &eth_addr,
         &TokenAmount::from_whole(10_000),
         METHOD_SEND,
         None::<RawBytes>,
     );
-    let account = v.normalize_address(&eth_addr).unwrap();
-    let mut actor = v.get_actor(&account).unwrap();
+
+    // FIXME: refactor this behaviour in terms of the abstract VM trait
```

Comment:
Don't commit `FIXME`. Is this just the same issue as the comment on the function signature? Can omit the comment here (move it to the signature) if so.